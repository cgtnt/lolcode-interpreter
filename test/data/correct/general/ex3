HAI 

I HAS A numone ITZ 5
I HAS A numtwo ITZ 3

I HAS A total ITZ SUM OF numone AN numtwo 

VISIBLE SMOOSH "TOTAL IZ " AN total MKAY!

KTHXBYE
