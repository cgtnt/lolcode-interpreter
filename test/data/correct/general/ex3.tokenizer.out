@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_DECLARE_VAR:I HAS A-@3:T_T_IDENTIFIER:numone-@3:T_DECLARE_SET_VAR:ITZ-@3:T_T_INT:5-@4:T_COMMAND_TERMINATOR:
-@4:T_DECLARE_VAR:I HAS A-@4:T_T_IDENTIFIER:numtwo-@4:T_DECLARE_SET_VAR:ITZ-@4:T_T_INT:3-@6:T_COMMAND_TERMINATOR:
-@6:T_DECLARE_VAR:I HAS A-@6:T_T_IDENTIFIER:total-@6:T_DECLARE_SET_VAR:ITZ-@6:T_PLUS:SUM OF-@6:T_T_IDENTIFIER:numone-@6:T_AND:AN-@6:T_T_IDENTIFIER:numtwo-@8:T_COMMAND_TERMINATOR:
-@8:T_WRITE_STDOUT:VISIBLE-@8:T_CONCAT:SMOOSH-@8:T_T_STRING:"TOTAL IZ "-@8:T_AND:AN-@8:T_T_IDENTIFIER:total-@8:T_END_INF:MKAY-@8:T_BANG:!-@10:T_COMMAND_TERMINATOR:
-@10:T_END:KTHXBYE-@11:T_COMMAND_TERMINATOR:
-@11:T_EOF:
